doc/ven_impl/ven_template/README.md

```diff
@@ -0,0 +1,40 @@
+# CNTT Vendor Implementation - VI
+
+## Vendor Name
+Vendor XX
+
+## Target Reference Architecture :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RA-1) |  v3.0   |   |
+
+## Target Reference Conformance :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RC-1) |  v3.0   |   |
+
+## VI Release Information
+
+**Release: _1.0-alpha_**
+
+**Version: _1.0_**
+
+**Release Date: _17th April 2020_**
+
+## Version History
+
+| Version | Date | Note
+| --- | --- | --- |
+| 1.0-alpha | TBD | TBD|
+
+## Content
+* [Cookbook](./cookbook.md)
```

Comment:
will every Vendor have their own cookbooks? If yes, then the cookbooks need Vendor Information

Reply:
Yes, but that information will be stored once in README.md insider vendor own folder.